lib/twilio/audiohelper.ts

```diff
@@ -424,11 +424,7 @@ class AudioHelper extends EventEmitter {
    * @private
    */
   _unbind(): void {
-    if (!this._mediaDevices || !this._enumerateDevices) {
-      throw new NotSupportedError('Enumeration is not supported');
-    }
```

Comment:
Yeah it's strange that we have this here. We should not have to throw when cleaning this up.